docsrc/README.md

```diff
@@ -0,0 +1,22 @@
+sphinx
+======
+
+## Getting started - building from source.
+1. Follow the `Project Setup` from the Readme.mp from the root of the project. 
+
+2. To build the content run `build clean` to remove existing files then run `build gethub' to generate the 
+pages which will be saved in the donar-docs-pipeline/docs folder. 
```

Comment:
there is a consistent typo of `donar` instead of `donor`

Reply:
fixed